modules/ssl-certificate/tests/alternative_domain/1-example.tf

```diff
@@ -0,0 +1,7 @@
+module "this" {
+  source = "../.."
+
+  validate            = false
+  domain              = "*.devops.dasmeta.com"
+  alternative_domains = ["dm.example.io"]
```

Comment:
did the issue solved also in case you have more than one alternative domain? can you try?